init_database.sh

```diff
@@ -1,2 +1,2 @@
-sudo su -c "createuser -sdrPW $1" postgres
-sudo su -c "createdb -h localhost -p 5432 -U $1 taskbox" postgres
+sudo -u postgres createuser -sdrPW $1
+sudo -u postgres createdb -h localhost -p 5432 -U $1 poopbox
```

Comment:
```suggestion
sudo -u postgres createdb -h localhost -p 5432 -U $1 taskbox
```

Love it but I assume this is what you meant?